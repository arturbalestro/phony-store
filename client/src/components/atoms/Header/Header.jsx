import React from "react";

import { StyledHeader } from "../../../styles/StyledHeader";
import logo from "../../../img/product-logo.png";
import { locales } from "../../../utils/locales";

const Header = () => (
  <StyledHeader>
    <img className="logo" src={logo} alt="Phony Store Logo" />
    <h1>{locales.mainTitle}</h1>
    <h3>{locales.mainSubTitle}</h3>
  </StyledHeader>
);

export default Header;
