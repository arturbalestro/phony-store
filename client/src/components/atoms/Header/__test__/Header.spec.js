// @flow
import Header from '../Header';
import React from 'react';
import renderer from 'react-test-renderer';

test('<Header /> renders correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
