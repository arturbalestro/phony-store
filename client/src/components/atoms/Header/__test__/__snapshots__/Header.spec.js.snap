// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<div
  className="sc-AxjAm dNblVa"
>
  <div
    className="top-container"
  >
    <a
      href="/"
    >
      <img
        alt="Phony Store Logo"
        className="logo"
        src="product-logo.png"
      />
    </a>
  </div>
</div>
`;
